resources/views/users/index.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layouts.master')
+
+@section('main-content')
+
+    @include('flash::message')
+
+    @if(!isset($users))
```

Comment:
if the $users is null, the view just view `Nothing to show!` in if statement, it easy to read the code, i think the positive have the same thing, but the line `Nothing to show!` in the bottom, it so hard to view the code!